semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/http/HttpSemanticConventionUtils.java

```diff
@@ -202,6 +203,8 @@ public static boolean isAbsoluteUrl(String urlStr) {
     } else if (SpanSemanticConventionUtils.isServerSpanForOtelFormat(attributeValueMap)
         || SpanSemanticConventionUtils.isServerSpanForOCFormat(attributeValueMap)) {
       return getHttpUrlForOtelFormatServerSpan(attributeValueMap);
+    } else if (attributeValueMap.containsKey(HTTP_URL.getValue())) {
```

Comment:
the above two cases would return empty optional, so the final case would not be reachable

Reply:
resolved